docs/stack_demo.md

```diff
@@ -0,0 +1,104 @@
+# How to run Demo version of Scarlet
+
+## Stack Version
+
+Development on this sprint landed on the `deblender_sprint` branch. The demo output was run using the `w_2018_19` shared stack on lsst-dev + 5 local repos:
+
+```[yusra@lsst-dev01]$ eups list -s | grep LOCAL
+autograd              LOCAL:/project/yusra/lsst_devel/autograd/_eupspkg/binary/autograd/master-g6a9ef8c4bf      setup
+meas_extensions_scarlet LOCAL:/project/yusra/lsst_devel/meas_extensions_scarlet         setup
+pipe_drivers          LOCAL:/project/yusra/lsst_devel/pipe_drivers      setup
+pipe_tasks            LOCAL:/project/yusra/lsst_devel/pipe_tasks        setup
+scarlet               LOCAL:/project/yusra/lsst_devel/scarlet/_eupspkg/binary/scarlet/deblender_sprint-gc57351e96f      setup
+```
+
+To setup a demo stack:
+
+```bash
+ source /software/lsstsw/stack/loadLSST.bash 
+ source scl_source enable devtoolset-6
+ source scl_source enable rh-git29
+ setup lsst_distrib -t w_2018_19
+ 
+ git clone https://github.com/lsst-dm/autograd
+ cd autograd
+ # rm -r _eupspkg if you need to rebuild
+ eupspkg -e -v 1 fetch
+ eupspkg -e -v 1 prep
+ eupspkg -e -v 1 config
+ eupspkg -e -v 1 build
+ eupspkg -e -v 1 install
```

Comment:
Need a `setup -r _eupspkg/<something>` here (and similar below).

Reply:
Also, I think you *only* need the `install` line; that will invoke the rest of the `eupspkg` ops automatically.